Gruntfile.js

```diff
@@ -5,26 +5,24 @@ module.exports = function(grunt) {
 
   var tsConfig = {
     dev: {
-      src: ["src/**/*.ts", "typings/**/*.d.ts"],
-      outDir: "build/src/",
+      src: ["src/**/*.ts"],
+      out: "plottable.js",
       options: {
         target: "es5",
         noImplicitAny: true,
         sourceMap: false,
         declaration: true,
-        compiler: "./node_modules/typescript/bin/tsc",
```

Comment:
As long as our package.json has a specified typescript package, it's fine.  https://github.com/TypeStrong/grunt-ts/issues/280 Also, `grunt-ts` is meaning to upgrade likely today, so maybe we don't even have to worry about this.